cobalt/browser/BUILD.gn

```diff
@@ -160,6 +160,7 @@ static_library("browser") {
     "//cobalt/browser/memory_tracker:memory_tracker_tool",
     "//cobalt/browser/metrics",
     "//cobalt/build:cobalt_build_id",
+    "//cobalt/build:cobalt_build_info",
```

Comment:
Does anything in the cobalt build tree depend on the output file being created? If not, what is the dependency? We want to make sure it's listed as a dependency in the right place

Reply:
Nothing depends on these files in Cobalt. I put it here because I needed something to run the action whenever Cobalt is built. There are downstream Buildbot dependencies on the file being generated.

Reply:
Can we move it to a higher level target? i.e. a gn_all target could be a good place

Reply:
If this is hard to do because of existing targets we build in CI being too specific dw about it